esp-wifi/examples/coex.rs

```diff
@@ -75,11 +75,19 @@ fn main() -> ! {
     controller.start().unwrap();
     println!("is wifi started: {:?}", controller.is_started());
 
-    println!("Start Wifi Scan");
-    let res: Result<(heapless::Vec<AccessPointInfo, 10>, usize), WifiError> = controller.scan_n();
-    if let Ok((res, _count)) = res {
-        for ap in res {
-            println!("{:?}", ap);
+    #[cfg(not(esp32c2))]
+    {
+        // on ESP32-C2 this will consume too much stack memory making the example crash
+        use embedded_svc::wifi::AccessPointInfo;
+        use esp_wifi::wifi::WifiError;
+
+        println!("Start Wifi Scan");
+        let res: Result<(heapless::Vec<AccessPointInfo, 10>, usize), WifiError> =
```

Comment:
Is this still a problem if we reduce the number of results to 2?

Reply:
Maybe we just remove this in the coex example for all chips? We already have it in DHCP ones etc?

Reply:
Yes - will do that 👍